roles/nova-data/tasks/monitoring.yml

```diff
@@ -13,6 +13,7 @@
   sensu_metrics_check: name=check-cinder-sessions plugin=check-cinder-sessions.py
                        interval=10
   notify: restart sensu-client
+  when: "'rbd_volumes' not in cinder.enabled_backends"
```

Comment:
That's fine, commas are just part of the string. The search filter will look for the contiguous block of chars `rdb_volumes`.